### PR TITLE
Fb profile badge

### PR DESCRIPTION
**Pull request by dingepfl**
This PR adds Badges for the reputation system, they are displayed on the profile of the user and the comment of each user under an Offer has a background color corresponding to his badge level (bronze, silver or gold)

### CODE REVIEW

**Comment by dingepfl** [comment]
The badge images are quite low quality and pixelated right now as they are mostly placeholder items.
I will update them with better quality ones in the near future.

File: app/src/main/java/ch/epfl/sweng/swenggolf/profile/User.java [hunk 1]
```diff
@@ -24,7 +24,7 @@
     private String photo;
     private String preference;
     private String description;
-    private int points;
+    private int points = 0;
```
Comment (ejolles) [inline app/src/main/java/ch/epfl/sweng/swenggolf/profile/User.java]: The points are initialized twice, either we keep the "DEFAULT_POINTS=0" either we keep your "points = 0".
You can choose!

Comment (dingepfl) [reply app/src/main/java/ch/epfl/sweng/swenggolf/profile/User.java]: You are absolutely right, I didn't pay attention to that...
I will correct it right away!

Comment (dingepfl) [reply app/src/main/java/ch/epfl/sweng/swenggolf/profile/User.java]: The `DEFAULT_POINTS = 0` solution seems cleaner to me and more consistent with the general coding style of the app

**Pull request merged**